notary-api/src/jobs/payments.js

```diff
@@ -7,7 +7,7 @@ import { getDataOrder } from '../operations/dataExchange';
 import { notarizationResults } from '../utils/stores';
 
 /**
- * BatPay.Transfer handler: TODO: addUnlockJob
+ * BatPay.registerPayment handler: TODO: addUnlockJob
```

Comment:
Missed line 23
`const notarization = await notarizationResults.safeFetch(transfer.lock);`
should be 
`const notarization = await notarizationResults.safeFetch(transfer.lockingKeyHash);`
test are failng due to this.

Reply:
Should we rename `sendUnlock` to  `sendUnlockPayment` ?